wiki/People/The_Team/Beatmap_Nominators/ru.md

```diff
@@ -0,0 +1,251 @@
+---
+tags:
+  - BN
+  - BNG
+  - БН
+  - Номинатор
+---
+
+# Номинаторы
+
+*См. также: [правила для номинаторов](Rules), [ожидания от номинаторов](Expectations), [общая информация](General_Information), [как стать номинатором](Becoming_a_Beatmap_Nominator)*
+
+**Номинаторы** (также ***BN***, ***БН***) — группа пользователей, занимающихся [номинированием карт на ранк](/wiki/Beatmap_ranking_procedure). На форуме и в обсуждениях карт их имена имеют фиолетовый цвет.
+
+## Роли и обязанности
+
+Номинаторы отвечают за [моддинг](/wiki/Modding) карт в категории [Pending](/wiki/Beatmaps#ожидающие) и их дальнейшее продвижение по процедуре ранкинга. К ним стоит обращаться за советом или просьбами, связанными с моддингом и маппингом. В возможности номинаторов входит [накладывание вето](Beatmap_Veto) на карты, которые не нарушают критерии ранкинга, но, по их мнению, имеют проблемы, которые перед ранком нужно решить.
+
+В дополнение к этому, номинаторы должны соответствовать [ожиданиям](Expectations) — набору рекомендаций по тому, как вести деятельность и помогать другим мапперам и сообществу в целом.
+
+Номинаторы делятся на две группы: полноправные (Full Beatmap Nominators) и на испытательном сроке (Probationary Beatmap Nominators).
+
+### Полноправные номинаторы
+
+Помимо регулярного моддинга и ранкинга карт, полноправные номинаторы могут дисквалифицировать карты в категории Qualified, содержащие ошибки или недочёты. Они также могут участвовать в отборе заявок в номинаторы, если заинтересованы в продвижении в NAT ([команду оценки номинаций](/wiki/People/The_Team/Nomination_Assessment_Team)).
+
+Примерно раз в три месяца деятельность полноправных номинаторов проверяется NAT. Если кто-то из номинаторов ведёт себя неподобающе или номинирует карты, которые потом часто дисквалифицируют, его отправляют на испытательный срок. Этого не происходит с номинаторами, которые редко модят — вместо этого их со временем исключают из команды, если их активность остаётся на низком уровне.
+
```

Comment:
this header has 3 paragraphs in the original, might be outdated?

Reply:
I merged the last two into one